src/GovukNotify/Client/BaseClient.cs

```diff
@@ -34,55 +35,71 @@ public BaseClient(IHttpClient client, string apiKey, string baseUrl = "https://a
             this.client.BaseAddress = ValidateBaseUri(BaseUrl);
             this.client.AddContentHeader("application/json");
 
-            var productVersion = typeof(BaseClient).GetTypeInfo().Assembly.GetName().Version.ToString();
+            var productVersion = typeof(BaseClient).GetTypeInfo().Assembly.GetName().Version?.ToString();
```

Comment:
This change is required as there was the possibility of a `NullReferenceException` in the original code - the `Version` attribute may be null.

If nullable types were enabled for the project it would likely help catch other similar problems.